README.md

```diff
@@ -393,6 +393,27 @@ The bundled connectors are registered and discovered using this mechanism via
 entries in the `pyproject.toml` file under the
 `[tool.poetry.plugins."servo.connectors"]` stanza.
 
+### Writing your own Connector class
+
+#### Exporting the Connector
+If you're writing your own connector in an external package, you need to include
+```yaml
+[tool.poetry.plugins."servo.connectors"]
+"my_connector" = "my_project.foo:MySpecialConnector"
+```
+in your `pyproject.toml` - you don't need to change the `pyproject.toml` in the servox project.
+This is sufficient for the servo to discover your connector and make it available in `servo.yaml`
+for configuration under the `myspecial:` key. Note that the configuration key is reflected from the
+lexical token representing the class name (e.g. `MySpecialConnector` in this example). To use a
+different connector name, see the following section.
+
+#### Notes on Connector Discovery
+The `connector_routes` loaded by the `servo/connector.py:ConnectorLoader` will use the `echo
+MyConnector | sed 's/Connector//' | lowercase` name for the top-level key in `servo.yaml` unless you
```

Comment:
This explanation is missing the component which converts the CamelCase names to snake_case here: https://github.com/opsani/servox/blob/796946547c97e513780b6a6175d2edbd9bd0252d/servo/connector.py#L311